docs/PowerAuth-2023.08.md

```diff
@@ -0,0 +1,71 @@
+# PowerAuth 2023.08
+
+## Migration guides
+
+For updating to 2023.08, please follow these migration guides:
+
+- [PowerAuth Server - Migration from version 1.4.0 to version 1.5.0](https://github.com/wultra/powerauth-server/blob/develop/docs/PowerAuth-Server-1.5.0.md)
+- [PowerAuth Push Server - Migration from version 1.4.0 to version 1.5.0](https://github.com/wultra/powerauth-push-server/blob/develop/docs/PowerAuth-Push-Server-1.5.0.md)
+- [PowerAuth Web Flow - Migration from version 1.4.0 to version 1.5.0](https://github.com/wultra/powerauth-webflow/blob/develop/docs/Web-Flow-1.5.0.md)
+
+## Components for version 2023.08
+
+### Back-End Applications
+
+| Component | Application Name | Version | Description |
+|---|---|---|---|
+| PowerAuth Server | `powerauth-java-server.war` | 1.5.0 | Core back-end component for PowerAuth stack. |
+| PowerAuth Admin | `powerauth-admin.war` | 1.5.0 | Administration console for PowerAuth Server. |
+| PowerAuth Push Server | `powerauth-push-server.war` | 1.5.0 | Simple to deploy push server for APNS and FCM. |
+| Enrollment Server | `enrollment-server.war` | 1.5.0 | Enrollment server for PowerAuth. |
+| PowerAuth Web Flow | `powerauth-webflow.war` | 1.5.0 | Central web authentication page. |
+| PowerAuth Next Step | `powerauth-next-step.war` | 1.5.0 | Authorization server used for PowerAuth Web Flow component. |
+| PowerAuth Data Adapter | `powerauth-data-adapter.war` | 1.5.0 | Customization component for PowerAuth Web Flow. |
+| PowerAuth Tpp Engine | `powerauth-tpp-engine.war` | 1.5.0 | Third party provider registry and consent engine. |
+
+### Utilities
+
+| Component | Application Name | Version | Description |
+|---|---|---|---|
+| PowerAuth Command Line Tool | `powerauth-java-cmd.jar` | 1.5.0 | Command line tool for integration testing. |
+
+### Mobile Libraries
+
+| Platform | Package Name | Version | Description |
+|---|---|---|---|
+| iOS | `PowerAuth2` | 1.8.0 | A client library for iOS. |
+| watchOS | `PowerAuth2ForWatch` | 1.8.0 | A limited library for watchOS. |
+| iOS App Extensions | `PowerAuth2ForExtensions` | 1.8.0 | A limited library for iOS App Extensions. |
+| Android | `com.wultra.android.powerauth:powerauth-sdk` | 1.8.0 | A client library for Android. |
+| React Native | `react-native-powerauth-mobile-sdk` | 2.4.1 | React Native wrapper library for PowerAuth. | 
+
+### Back-End Integration Libraries
+
+| Component | Library Name |  Version | Description |
+|---|---|---|---|
+| PowerAuth RESTful Model | `powerauth-restful-model.jar` | 1.5.0 | Model classes for request and response objects used in PowerAuth Standard RESTful API. |
+| PowerAuth RESTful API Security for Spring | `powerauth-restful-security-spring.jar` | 1.5.0 | High-level integration libraries for RESTful API security, build for Spring MVC. |
+| PowerAuth SOAP Client for Spring WS | `powerauth-java-client-spring.jar` | 1.5.0 | SOAP service client for PowerAuth Server service, built using Spring WS. |
```

Comment:
We have dropped the SOAP support, haven't we?

Reply:
Good point, should have been migrated to REST long time ago.